terraform/modules/s3/s3.tf

```diff
@@ -0,0 +1,35 @@
+# Creates S3 bucket for nwp data
+
+# make sure all public access is blocked
+# https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_public_access_block
+resource "aws_s3_bucket_public_access_block" "s3-nwp-block-public-access" {
+  bucket = aws_s3_bucket.s3-nwp-bucket.id
+
+  block_public_acls       = true
+  block_public_policy     = true
+  restrict_public_buckets = true
+  ignore_public_acls      = true
+
+}
+
+# Documentation: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket
+resource "aws_s3_bucket" "s3-nwp-bucket" {
+  bucket = "nowcasting-nwp-${var.environment}"
+  acl    = "private"
+
+    lifecycle_rule {
+      id = "remove_old_files"
+      enabled = true
+
+      prefix = "data/"
+
+      expiration {
+          days = 180
```

Comment:
change to 30 days

Reply:
The idea is to just keep 30 days for inference. The backtesting is down with a different pipeline at the moment. Can easily change this